documentation/cluster-controller/cluster-controller.adoc

```diff
@@ -0,0 +1,296 @@
+= Cluster Controller
+
+The cluster controller is in charge of deploying a Kafka cluster alongside a Zookeeper ensemble. It's also able to deploy a
+Kafka Connect cluster which connects to an existing Kafka cluster. Such a cluster can be deployed using the Source2Image OpenShift
+feature as well in order to include more connectors plugins.
+
+When the cluster controller is up and running, it starts to "watch" for ConfigMaps containing the Kafka or Kafka Connect
```

Comment:
just "up", "up and running" is a bit tautologous.

Reply:
So all the book I read "XXXX up and running" are wrong ? :P

Reply:
It's not wrong, but it's a little informal. In a technical document, like this, people interpret words with a technical context, and in this context what is the difference between "up" and "running"? The cluster controller can't it be up without also being in a running state, so it's clearer to just use one term.